src/model.py

```diff
@@ -2144,9 +2146,8 @@ def forward(self, batch):
         for i in range(1, shared_layers):
             sentence_output = shared[f"cnn_{i}"](sentence_output)
 
-            # update word embeddings with shared features learned from this cnn layer
-            embeddings = torch.cat((embeddings, sentence_output), dim=1) # cat embedding dim
-            # FIXME embeddings is never even used..?
+        # update word embeddings with shared features learned from this cnn layer
+        embeddings = torch.cat((embeddings, sentence_output), dim=1) # cat embedding dim
```

Comment:
Reason this was excluded was embeddings was only being updated during higher level shared. However, it should be updated no matter what. 

Does this work when shared layers = 0 or 1? It should, since the sizes never change depending on layer count.